### PR TITLE
[RI1] Fill development validations

### PR DESCRIPTION
**Pull request by collivier**
It explains the development process already in place in RI Gates
and OPNFV more generally.

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_impl/cntt-ri/chapters/chapter07.md [hunk 1]
```diff
@@ -71,18 +72,67 @@ Details explaining:
...
+to achieve the stability needed by CNTT. Then the deployment validations
+previously detailed must be also applied for every patch proposed in RI and
+all changes published in [Gerrit](https://gerrit.opnfv.org/) must be
+automatically voted -1 by Jenkins in case of failures.
+
+Only the following test cases must pass as a temporarily bypass in
+[RI gates](https://build.opnfv.org/ci/view/airship/job/airship-latest-gate/) to
+allow merging the next patches. At the time writing, CNTT RI is not compliant
+with [RA1 Core OpenStack Services APIs]({{ "/doc/ref_arch/openstack/chapters/chapter05.html" | relative_url }})
+and then a part of the containers selected in [RC1 TC Requirements]({{ "/doc/ref_cert/lfn/chapters/chapter09.html" | relative_url }})
```
Comment (mf4716) [inline doc/ref_impl/cntt-ri/chapters/chapter07.md]: Relative URL paths do not appear to be working.  Can you resolve?

Comment (collivier) [reply doc/ref_impl/cntt-ri/chapters/chapter07.md]: It works. It's postprocessed automatically and changed to the right path. That's part of the best practices to manage url prefix in Jekyl (https://cntt-n.github.io/CNTT)

Comment (mf4716) [reply doc/ref_impl/cntt-ri/chapters/chapter07.md]: If there's a way to preview, let me know.  I can preview with markdown, using either StackEdit or GitHub.  Not able to preview / resolve your URL.  Regardless, I'll approve, and we can resolve broken URLs in subsequent patches.  The content is good.

**Comment by collivier** [comment]
@mf4716 Thank you for the quick approval.
@rabiabdel @fuqiao123 would you mind reviewing and merging it before the release? It's about the work already in place https://build.opnfv.org/ci/view/cntt/job/cntt-latest-gate/

**Comment by rabi-abdel** [comment]
one more approval is needed for this to be merged. in case there was no approval received before Code Freeze, I will merge anyway.

**Pull request merged**